manifests/fedora-coreos.yaml

```
@@ -33,6 +33,9 @@ conditional-include:
     # Remove this once we have Podman v5 in f39.
     # xref: https://github.com/coreos/fedora-coreos-tracker/issues/1629
     include: cni-plugins.yaml
+  - if:
+      - releasever >= 41
```

```suggestion
  - if: stream == "rawhide"
```

Let's actually match against `rawhide` here. Eventually F41 will come to a prod stream. If we choose to turn composefs on for prod streams we should open a ticket for it and record the results of the discussion and then make the change.